platforms/windows/src/util.rs

```diff
@@ -4,7 +4,7 @@
 // the LICENSE-MIT file), at your option.
 
 use std::{
-    convert::{From, TryInto},
+    convert::{From, Into, TryInto},
```

Comment:
`From` and `Into` are already in the prelude, no need to import them here.

```suggestion
    convert::TryInto,
```